src/Dfc.CourseDirectory.Core/Dfc.CourseDirectory.Core.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="HtmlAgilityPack" Version="1.11.22" />
     <PackageReference Include="HtmlSanitizer" Version="5.0.274-beta" />
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.4.0" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.3" />
```

Comment:
Consider using `Azure.Storage.Blobs` library (I believe this is the latest/recommended API to use)